content/workshops/agile-scrum/_index.md

```diff
@@ -1,9 +1,8 @@
 ---
 date: 2019-03-19T16:50:16+02:00
 title: Agile and Scrum
+Ready: True
```

Comment:
little r for ready. I don't think this will work

and weight isn't really required. It specified the sort order of links in the left hand side of the page
we also don't use date.

Reply:
edited, should be fine now changed it to little r and little t